release-calendar/src/server/repository-service.ts

```diff
@@ -62,14 +62,6 @@ export class RepositoryService {
   }
 
   async getCurrentReleases(): Promise<Promotion[]> {
-    const rollbackQuery = this.promotionRepository
-      .createQueryBuilder('promotion')
-      .where('promotion.channel = :channel', {channel: Channel.ROLLBACK})
-      .select('promotion.releaseName')
-      .groupBy('promotion.releaseName')
-      .addSelect('promotion.channel')
-      .getMany();
-
     const channelQueries = [
```

Comment:
Now that all the Channel values are included here, you can do:
```typescript
const channelQueries = Object.keys(Channel).map((eachChannel) => {
    ...
```

Or even
```typescript
return Promise.all(Object.keys(Channel).map((eachChannel) => {
    ...
```

Reply:
Supernit: if you rename the anonymous function's argument from `eachChannel` to `channel`, you can change the `.where` line to `.where('promotion.channel = :channel', {channel})`